engine/Dockerfile

```diff
@@ -9,12 +9,10 @@ RUN pip install -r requirements.txt
 
 COPY ./ ./
 
-# Collect static files and create an SQLite database
 RUN mkdir -p /var/lib/oncall
-RUN DJANGO_SETTINGS_MODULE=settings.prod_without_db DATABASE_TYPE=sqlite3 DATABASE_NAME=/var/lib/oncall/oncall.db SECRET_KEY="ThEmUsTSecretKEYforBUILDstage123" python manage.py collectstatic --no-input
```

Comment:
Looks like removing that breaks something in the docker build.  On another note we should probably add a step to the Github workflow to run that check instead of only in drone or run the drone pipeline on PRs not just merge to dev.

Reply:
good catch/suggestion.

got around the failing Docker image build in [d85e975](https://github.com/grafana/oncall/pull/671/commits/d85e975105198e425572adfced2efdb6608a0ac5) and added a job to the CI build to build the engine docker image in [f6cb9ed](https://github.com/grafana/oncall/pull/671/commits/f6cb9eda71aaaf63f80c419fa8fa33fc2fc755a1)